src/main/java/org/codehaus/mojo/license/AbstractLicenseNameMojo.java

```diff
@@ -114,7 +114,7 @@ public abstract class AbstractLicenseNameMojo extends AbstractLicenseMojo {
      * @parameter property="license.licenseName"
      * @since 1.0
      */
-    @Parameter(property = "license.licenseName")
+    @Parameter(property = "license.licenseName", required = true)
```

Comment:
We can also simplify method - `getLicense`

Reply:
Thanks, good point.

* Remove empty check from getLicense(...) as we have same check in the license repository.
* Replace `isEmpty` with `isBlank` because a user can set blank name through a command line parameter. It fails anyway but with `isBlank` the error message better describes what the issue is.

Reply:
As I know required parameter cannot be null nor blank